FfiSpike/Declarations.h

```diff
@@ -69,7 +69,7 @@ struct DeviceData {
 	DeviceInfo* device_info;
 	std::map<const char*, HANDLE> services;
 	int sessions;
-	afc_connection* afc_conn_p;
+	std::map<const char*, afc_connection*> afc_connections;
```

Comment:
I'd prefer the keys of this map be of type `std::string` instead of `const char*` because we can pass those around by reference.